types/koa/index.d.ts

```diff
@@ -512,6 +513,12 @@ declare class Application<
      * @api private
      */
     onerror(err: Error): void;
+
+    /**
+     * Make HttpError available to consumers of the library so that consumers don't
+     * have a direct dependency upon `http-errors`
+     */
+    static HttpError: typeof HttpError;
```

Comment:
can this be an export of the `Application` namespace?

this would allow it to be imported as

```ts
import { HttpError } from 'koa';
// HttpError

import * as Koa from 'koa';

// Koa.HttpError
```

---

this would be consistent with the existing exports for `Middleware`, `Context`, etc.

Reply:
I agree, this is much better.